packages/x-components/tests/e2e/specs/cucumber/search-box.spec.ts

```diff
@@ -69,15 +68,6 @@ And(
 );
 
 // Scenario 2
-And('the number of next query results are stored', () => {
-  if (cy.$$('[data-test = "next-queries"]').length > 0) {
```

Comment:
4 hours later, I finally managed to test that this implementation for this step looks like it works.

```
  cy.get('body').then($body => {
    nextQueriesResults = $body.find('[data-test="next-query"]').length;
  });
```

Reply:
Which somehow is the [recommended approach](https://docs.cypress.io/guides/core-concepts/conditional-testing#Element-existence) for this cases.